src/library/scala/Predef.scala

```diff
@@ -304,9 +302,38 @@ object Predef extends LowPriorityImplicits with DeprecatedPredef {
 
   // printing -----------------------------------------------------------
 
+  /** Prints an object to `out` using its `toString` method.
+    *
+    *  @param x the object to print; may be null.
+    */
```

Comment:
The asterisk should align with the first of the double stars. See the top comment. Javadoc style (see second comment in source) seems to sneak by, but alt-Scaladoc style is out of favor. I'm not the comment cop, but since this is `Predef`, uniformity is worth enforcing. Is there a SO Q&A, "How many ways are there to write Scaladoc comments in Scala?" Luckily, April 1 is just around the corner.